iglo/league/utils/paring.py

```diff
@@ -26,18 +26,65 @@ def round_robin(n: int) -> Pairing:
         return list(_round_robin_odd(d, n))
 
 
+def banded_round_robin(player_count: int, band_size: int, add_byes: bool = True) -> Pairing:
```

Comment:
in what scenario add_byes =False is sensible?

Reply:
Byes were needed and added for custom scoring. They were not present earlier and they are not needed for algorithm correctness. In a way this is 2 algos in one, and we don't use other. It is like a library.
I was planning to writeup this algo on arxiv (you can't find it online), and then it would be first without byes.

I guess now this is a nice documentation that the algorithm works with add_byes=False. 
I believe it helps when one tries to read it.

I you feel that it is detrimental to the readability I can hardcode it.